### ISSUE TITLE
Inconsistant spaces (minute vs hour)

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#1203]
```+  <plurals name="minutes">
+    <item quantity="one"> minute </item>
+    <item quantity="other"> minutes</item>
+  </plurals>
+  <plurals name="hours">
+    <item quantity="one"> hour</item>
+    <item quantity="other"> hours</item>
+  </plurals>
+  <plurals name="days">
+    <item quantity="one"> day</item>
+    <item quantity="other"> days</item>
+  </plurals>```

**Comment by mostafa07** [comment]
I will start working on it.

**Comment by mostafa07** [comment]
@teolemon could you please check this?
I have fixed the typo in PR #1209

**Comment by mostafa07** [comment]
#fixed in PR #1209
